### ISSUE TITLE
Alt text shown in HTML

### ISSUE THREAD

**Issue by jszh** [arXiv/html_feedback#3618]
### Description

\Description is the specified way in ACM's paper template to include figure alt text. They are not processed properly and show up in HTML.

### (Required) What is this issue most closely related to? Select one.

Choose One

### Internal issue ID

db48e469-2525-4a79-9435-17803068d767

### Paper URL

https://arxiv.org/html/2504.02110v1

### Browser

Chrome/133.0.0.0

### Device Type

Desktop

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.
